lib/utils/logger/log_settings.js

```diff
@@ -1,103 +1,96 @@
-const lodashClone = require('lodash.clone');
-
-const Settings = {
-  outputEnabled: true,
-  showResponseHeaders: false,
-  showRequestData: {
-    enabled: true,
-    trimLongScripts: true
-  },
-  detailedOutput: true,
-  disableErrorLog: false,
-  log_timestamp: false,
-  timestamp_format: null,
-  enabled: true
-};
-
-module.exports = new (function() {
-  const logSettings = lodashClone(Settings, true);
-
-  class LogSettings {
-    get outputEnabled() {
-      return logSettings.outputEnabled;
-    }
-
-    get detailedOutput() {
-      return logSettings.detailedOutput;
-    }
+class LogSettings {
+  constructor() {
+    this.__outputEnabled = true;
```

Comment:
Instead of `__outputEnabled`, we should use `#outputEnabled`.
Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields

Reply:
I have no problem with either notations, it's just that `__` notation is what we're using everywhere in the Nightwatch project, and because we are anyway going to convert this file to TS just after this PR is merged where we'll use the `private` keyword, using/not using `#` doesn't really matter.

Reply:
Done. Needed to update eslint to work with # as well. But not sure if # is supported in Node.js 16, let's see.

Reply:
Okay, it seems that they are available since Node.js 12, but I wonder why it was only implemented in ECMAScript 2022?